handler/schedule.go

```diff
@@ -79,63 +79,77 @@ func (s *ScheduleHandler) createJobList() scheduler.JobList {
 		Jobs:   make([]scheduler.Job, 0),
 	}
 
-	if s.schedule.Spec.Archive != nil {
-		s.schedule.Spec.Archive.ArchiveSpec.Resources = s.mergeResourcesWithDefaults(s.schedule.Spec.Archive.Resources)
+	if archive := s.schedule.Spec.Archive; archive != nil {
```

Comment:
I've discussed this if cascade here with @ccremer already.

I feel like it's now the right time to try to get rid of this. WDYT?

Reply:
A generic solution – one that's not just some hack on top – is not trivial unfortunately. I've tried before and tried this morning, but was not able to come up with a nice solution. I'm open to suggestions :)

Reply:
Okay then let's move it into its own issue then.